src/test/README_TESTS

```diff
@@ -0,0 +1,22 @@
+Aby stworzyć zestaw testów jednostkowych, należy stworzyć nowy plik .js, w którym należy zaimportować funkcje i obiekty
+z TestUtils.js, następnie zdefiniować testy jednostkowe i na końcu stworzyć zestaw testów złożony z nich. Na końcu pliku
+należy wywołać na obiekcie zestawu testów metodę run().
+Ważne jest, aby funkcje używane przez testy jednostkowe zwracały zmienną logiczną, a najlepiej gdyby zwracały
+wyrażenie logiczne złożone z asercji stworzonych w TestUtils.js
+Przykład takiego pliku:
+
+
+
+var test = require("./TestUtils")
+
+var x1 = new test.UnitTest(()=>{return test.assertNotEqualsPrimitives(5,1)}, "5!=1 Test")
+var x2 = new test.UnitTest(()=>{return test.assertTrue(false)}, "Wrong Test")
+var suite = new test.TestSuite("Example");
+suite.addTest(x1);
+suite.addTest(x2);
+suite.run()
+
+
+
+Następnie w skrypcie runTests.py należy dodać do listy "suites" nazwy stworzonych plików bez rozszerzeń.
```

Comment:
Nie bardzo to eleganckie, ale roboczo będzie. To co możnaby lepiej to np. w skrypcie pythonowym podawać ścieżkę z której należy odpalić wszystkie testy. Gdyby też każdy test oznaczyć przez np. "--@Test" możnaby łatwo identyfikować funckcje testowe i suity zrobić z pythona per plik. Niemniej to tylko dywagacje, obecne rozwiązanie w zupełności wystarczy na nasze potrzeby